analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd

```diff
@@ -320,6 +326,28 @@ neurocytoma_results_df <- neurocytoma_results_df %>%
          Notes = if_else(!is.na(integrated_diagnosis), "Updated via OpenPBTA subtyping", Notes))
 ```
 
+### Handling Chordoma
+```{r}
+
+chordoma_results_df <- bind_exp_strategies(chordoma_results_df) %>%
+  inner_join(select(histologies_df,
```

Comment:
Looks like this notebook uses this group of variables a few times, might be easier to declare this as a vector too, in case you need to edit it in the future, then you will only need to edit in one place.

Reply:
good idea, thanks!